runtime/src/tokio.rs

```diff
@@ -23,7 +23,12 @@
 //! });
 //! ```
 
-use crate::{utils::Signaler, Clock, Error, Handle, Signal, METRICS_PREFIX};
+use crate::{
```

Comment:
Now that we have to use some traits to implement `serve_metrics`, it made sense to directly import all conflicting trait names with a `R-` prefix.